project/lang/class.default.php

```diff
@@ -40,7 +40,7 @@ class DefaultLang
 		'go_back' => "&lt;&lt; Back",
```

Comment:
Looks great !

One more question : Have you test all these change on your local machine ?

Reply:
all feature tested

Reply:
Ship it !